Neos.FluidAdaptor/Classes/Core/Widget/AbstractWidgetViewHelper.php

```diff
@@ -238,19 +239,15 @@ protected function initiateSubRequest()
 
                 /** @var $parentResponse ActionResponse */
                 $parentResponse = $this->controllerContext->getResponse();
```

Comment:
The "parentResponse" can be removed from the `ActionResponse` constructor above in Ln 216.